sips/sip-019/sip-019-token-metadata-update-notifications.md

```diff
@@ -212,6 +212,26 @@ Notifications that do not meet these requirements must be ignored.
 Given these constraints the notifications this SIP proposes should be taken as _hints_ to metadata
 indexers. Metadata indexers are not obliged to follow them.
 
+# Related work
+
+An alternative considered for token metadata update notifications is for them to be transmitted via
+an off-chain notification service such as an official mailing list or a forum post. While this
+channel would have had the advantage of being easier to update and faster to propagate, it has a few
+key disadvantages that make it inadequate for this SIP's intended use:
+
+1. It introduces a third party dependency
+    * An off-chain notification service would most likely be maintained by centralized entities
+      unrelated to the Stacks ecosystem which at any time may modify the channel, its reach, or its
+      rules.
```

Comment:
Besides the possibility of a single-point-of-failure notification service implementation going offline, I'm not sure why these other properties are inherently bad for a hinting service?

* Suppose a hint was sent in error, or triggered a bug in some big downstream indexers.  The ability to retroactively remove a hint to address this would be a good thing.

* Suppose a hint contained illicit data, or was used to dox someone or harass people (anyone can send them, so the payload contents really can be anything).  Wouldn't the ability to limit its reach and/or impose moderation rules prevent this use-case?

Reply:
The "third party dependency" argument itself isn't bad, though.  For example, you could make the case that separating notifications from the blockchain invites many different hinting service APIs and implementations, which puts us back at square one.  The act of pushing notifications into the blockchain imposes exactly one canonical way to access and store hints.

Reply:
Great points, thank you. Added your insights.